packages/clerk-js/src/core/clerk.ts

```diff
@@ -1530,7 +1531,7 @@ export default class Clerk implements ClerkInterface {
 
     this.#broadcastChannel?.addEventListener('message', ({ data }) => {
       if (data.type === 'signout') {
-        void this.handleUnauthenticated({ broadcast: false });
+        void this.handleUnauthenticated({ broadcast: true });
```

Comment:
Why do we need this?

Reply:
It's necessary to have broadcast to default to enable post the sign-out message to the BroadcastChannel, but it's not necessary to fill the `this. handleUnauthenticated` as `broadcast` prop is always true.